test/fitnesse/slim/converters/MapConverterTest.java

```diff
@@ -134,6 +134,13 @@ public void shouldRenderTableAsHtml() {
         "\t<tr class=\"hash_row\">", "\t\t<td class=\"hash_key\">c</td>", "\t\t<td class=\"hash_value\">d</td>", "\t</tr>", "</table>"), HtmlTag.endl), editor.toString(aMap()));
   }
 
+  @Test
+  public void nonHtmlStringWithPercentageReturnsEmptyMap() {
+    //See https://github.com/unclebob/fitnesse/issues/1190
+    MapConverter cnv = new MapConverter();
+    assertEquals(new HashMap<>(), cnv.fromString("1%"));
```

Comment:
Why don't you use `mapMap()` as done in other tests?

Reply:
Agree that's nicer. Didn't pay enough attention to how the other tests are implemented. Current revision is more consistent